src/scss/grommet-core/_objects.layer.scss

```diff
@@ -58,6 +58,10 @@
     }
   }
 
+  .#{$grommet-namespace}tiles {
```

Comment:
This is to fix `Tiles` infinite scroll within `Layer` component. https://github.com/grommet/grommet/pull/866#pullrequestreview-861943